### PR TITLE
Handle primary output disconnected case

### PR DESCRIPTION
**Pull request by kousiksatish**

### CODE REVIEW

File: native_src/display_utility_x11.cc [hunk 1]
```diff
@@ -187,17 +188,35 @@ RROutput DisplayUtilityX11::GetPrimaryRROutput()
...
-        unsigned int numberOfOutputs = 0;
-        RROutput *connectedOutputs = nullptr;
-        // If primary monitor is not set, but monitors are connected, first one is assumed as primary
-        if (this->TryGetConnectedOutputs(&numberOfOutputs, &connectedOutputs))
-        {
-            if (numberOfOutputs > 0) {
-                primaryRROutput = connectedOutputs[0];
+    unsigned int numberOfOutputs = 0;
+    RROutput *connectedOutputs = nullptr;
+
```
Comment (bavulapati) [inline native_src/display_utility_x11.cc]: When we handle the primary output case in getconnected outputs method, we can simply check if the list is empty and return 0 or the index 0 value.

Comment (kousiksatish) [reply native_src/display_utility_x11.cc]: Changed the logic

**Comment by kousiksatish** [comment]
Fixes #27

**Comment by kousiksatish** [comment]
Closes #27

**Pull request merged**